### PR TITLE
fix: frf211 added xml sitemap

### PR DESCRIPTION
**Pull request by RahatH07**
This is a static xml page, This xml file is for SEO

### CODE REVIEW

File: public/sitemap.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/cookie-policy</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/accessibility</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://www.nihr.ac.uk/</loc>
```
Comment (paddyduff31) [inline public/sitemap.xml:61]: Why is this in the sitemap?  It's a different site?

File: public/sitemap.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<!-- public/sitemap.xml -->
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk</loc>
+    <lastmod></lastmod>
```
Comment (paddyduff31) [inline public/sitemap.xml:6]: This `lastmod` tag should not be left blank, important for SEO.  Complete in this format:

YYYY-MM-DD

**Review by paddyduff31 — CHANGES_REQUESTED** [review]

File: public/sitemap.xml (hunk 1 above)
Comment (RahatH07) [reply public/sitemap.xml:61]: Its at the bottom of the homepage, As an image, I thought in a sitemap you would have to include every site is linked to. I can remove it no problem

File: public/sitemap.xml [hunk 3]
```diff
@@ -0,0 +1,64 @@
+<!-- public/sitemap.xml -->
```
Comment (paddyduff31) [inline public/sitemap.xml:1]: @RahatH07 looks like sonar has an issue with this line too, I know it's only a comment but can you remove too please?

File: public/sitemap.xml (hunk 2 above)
Comment (RahatH07) [reply public/sitemap.xml]: Added

**Review by paddyduff31 — APPROVED** [review]

**Review by kevinofficer — APPROVED** [review]

**Pull request merged**